crates/forge/src/main.rs

```diff
@@ -77,6 +83,9 @@ struct TestArgs {
     /// Run all tests regardless of `#[ignore]` attribute
     #[arg(long, conflicts_with = "only_ignored")]
     include_ignored: bool,
+
+    #[arg(value_enum, long, help = "Control when colored output is used", default_value_t = ColorOption::Always, value_name="WHEN")]
```

Comment:
You can just a comment starting with `///`, it will be parsed by `clap` and then you can remove the `help` section

Also the better description would be 

`Control whether colored output is used`

Reply:
`Control when colored output is used` is used in [`cargo test` documentation](https://doc.rust-lang.org/cargo/commands/cargo-test.html).

Should I change the description ?